gromacs/core.py

```diff
@@ -434,8 +434,8 @@ class GromacsCommand(Command):
 
     command_name = None
     driver = None
-    doc_pattern = """.*?(?P<DOCS>DESCRIPTION.*)"""
-    gmxfatal_pattern = """----+\n                   # ---- decorator line
+    doc_pattern = r""".*?(?P<DOCS>DESCRIPTION.*)"""
+    gmxfatal_pattern = r"""----+\n                  # ---- decorator line
```

Comment:
I am not 100% sure that the newline is actually captured by the raw-string expression.

Ought to check if there's _actually_ a test for that.

Reply:
Now there is.

(Of course it hadn't worked ... untested code is broken code!)